config/puma.rb

```diff
@@ -5,7 +5,7 @@
 preload_app!
 
 rackup      DefaultRackup
-port        ENV['PORT']     || 3000
+port        ENV['PORT']     || 80
```

Comment:
Did something necessitate this change?

Reply:
No, I accidentally checked in a local config change.